emails/long-time-without-login.pug

```diff
@@ -2,5 +2,5 @@ extends index
 block title
     h1 Hi, #{firstName}
 block description
-    p You’ve been inactive for too long. If you don’t log in within 7 days your account will be deleted.<br/>You’ll not be able to access your account once it’s deleted.
+    p You have been inactive for too long. If you dont log in within 7 days your account will be deleted.<br/>You will not be able to access your account once it is deleted.
```

Comment:
do not

Reply:
and change `subject` in index.js file too for this template

Reply:
fixed